compiler/tf2nnpackage-value-remote-test/CMakeLists.txt

```diff
@@ -0,0 +1,87 @@
+# Read "test.lst" if exists
+include("test.lst" OPTIONAL)
```

Comment:
1) other tests require `test.lst` and optionally read `test.local.lst`. how about using this name?
2) I don't see mandatory test list file... it's a bit strange. maybe add a NOTE about your intention.

Reply:
First, `test.lst` is mandatory because this test need `REMOTE_IP` and `REMOTE_USER`. But if I remove `OPTIONAL` keyword, configure step will stop. So, I set it to optional and return when there is no `REMOTE_IP` and `REMOTE_USER` below. 

Second, right there is no mandatory test list file this is why below some macro are empty. Actually, in `common-artifacts`, all of the modules `tf2nnpackage-value-remote-test` need to test - tflchef, tflite2circle, circle2circle, luci-interpreter - are tested. That is, `common-artifacts` already includes this modules except runtime test.

So, there is no need `test.lst`. In this module, only the runtime test is performed, and for this runtime test, generation of test data is required. And, `tcgenerate` in `${ARTIFACTS_SRC_PATH}/exclude.lst` means it won't generate test data, which is why below macro excludes specific opearators from runtime test.

Reply:
As talked off-line, we can add an empty `test.lst`, maybe with a comment at the first line.